pyproject.toml

```
@@ -58,6 +61,7 @@ lint = "pre-commit run --all-files"
 precommit = "pre-commit install"
 build = "docker build -t ghcr.io/python-discord/bot:latest -f Dockerfile ."
 push = "docker push ghcr.io/python-discord/bot:latest"
-test = "coverage run -m unittest"
+fast-test = "pytest -n auto"
```

How about this?

1. Nicer to start it with "test"
2. "nocov" is more meaningful than "fast" but perhaps no everyone will catch on to it meaning "no coverage"

```suggestion
test-nocov = "pytest -n auto"
```

---

I didn't particularly like "fast", but I didn't have a better name. I would usually just call this test, and have something indicating coverage in the other one, but I didn't want to break the existing convention.

nocov works, let's use that.